### PR TITLE
gcp - cloud run revision

### PR DESCRIPTION
**Pull request by evhenyshaliov**
Use cases:

``
policies:
  - name: gcp-run_revision_without_default_service_account
    description: |
      Cloud Run revision is configured with default service account
    resource: gcp.namespace-revision
    filters:
      - type: value
        key: spec.serviceAccountName
        op: contains
        value: -compute@developer.gserviceaccount.com
      - type: value
        key: status.conditions[?type=='Active'].status
        value:
          - "TRUE"
          - "True"
          - "true"
        op: intersect

### CODE REVIEW

File: tools/c7n_gcp/c7n_gcp/resources/cloudrun.py [hunk 1]
```diff
@@ -40,3 +40,21 @@ class resource_type(TypeInfo):
...
+    """GCP resource: https://cloud.google.com/run/docs/reference/rest/v2/projects.locations.services.revisions"""
+
+    class resource_type(TypeInfo):
+        service = "run"
+        version = "v1"
+        component = "namespaces.revisions"
+        enum_spec = ("list", "items[]", None)
+        scope_key = "parent"
+        scope_template = "namespaces/{}"
+        name = id = 'id'
```
Comment (kapilt) [inline tools/c7n_gcp/c7n_gcp/resources/cloudrun.py:56]: invalid id / name

File: tools/c7n_gcp/c7n_gcp/resources/cloudrun.py [hunk 2]
```diff
@@ -40,3 +40,21 @@ class resource_type(TypeInfo):
...
+        default_report_fields = ['displayName', 'expireTime']
```
Comment (kapilt) [inline tools/c7n_gcp/c7n_gcp/resources/cloudrun.py:57]: invalid report fields

File: tools/c7n_gcp/c7n_gcp/resources/cloudrun.py (hunk 1 above)
Comment (evhenyshaliov) [reply tools/c7n_gcp/c7n_gcp/resources/cloudrun.py]: sorry, fixed

File: tools/c7n_gcp/c7n_gcp/resources/cloudrun.py (hunk 2 above)
Comment (evhenyshaliov) [reply tools/c7n_gcp/c7n_gcp/resources/cloudrun.py]: thanks, fixed

**Review by kapilt — APPROVED** [review]
lgtm

**Pull request merged**